doc/texture_packs.txt

```diff
@@ -73,9 +73,9 @@ by texture packs. All existing fallback textures can be found in the directory
       when no texture was found
 
 * `object_crosshair.png`
-* the crosshair seen when pointing at an object. The settings
-`object_crosshair_color` and `object_crosshair_alpha` are used to create a cross
-when no texture was found
+    * the crosshair seen when pointing at an object. The settings
+    `object_crosshair_color` and `object_crosshair_alpha` are used to create a cross
+    when no texture was found
```

Comment:
```suggestion
    when no texture was found.
```

:D

Reply:
Please also fix this in L73 for consistency. :)

Reply:
Also fixed the past tense usage. How's that?

Reply:
I think it sounds better now. :+1: